packages/shared_preferences/shared_preferences/example/integration_test/shared_preferences_test.dart

```diff
@@ -96,23 +96,23 @@ void main() {
       preferences = await SharedPreferences.getInstance();
     });
 
-    tearDown(() {
-      preferences.clear();
+    tearDown(() async {
+      await preferences.clear();
       SharedPreferences.resetStatic();
     });
 
     runAllTests();
   });
 
-  group('setPrefix', () {
+  group('setPrefix', () async {
```

Comment:
Do we really want groups to be async?

Reply:
That was an unneeded tag anyway. I meant to erase it.